src/app/pages/AccountPage/Features/SendTransaction/index.tsx

```diff
@@ -42,7 +60,12 @@ export function SendTransaction() {
 
   return (
     <Box border={{ color: 'background-front-border', size: '1px' }} round="5px" background="background-front">
-      <Form>
+      <Form<SendTransactionFormState>
+        validate="submit"
```

Comment:
I went with full `Form` API.  Props like `validate` and `onSubmit` look good here. I am not sure about typed form, `onChange` and `value`.  Should I  discard these and  restore

```
  const [recipient, setRecipient] = useState('')
  const [amount, setAmount] = useState('')
  ```
pattern ?